wiki/Glossary/en.md

```diff
@@ -472,11 +471,13 @@ From the [ranking criteria](/wiki/Ranking_Criteria), the required [drain time](/
 
 ### M4M
 
-M4M, or _Mod for Mod_, is a term describing when a modder agrees to mod another modder's map in return for a mod on their own map.
+*Short for "mod for mod".*
+
+A term describing when a modder agrees to mod another modder's map in return for a mod on their own map.
 
 ### Marathon
 
-A marathon is a type of [beatmap](/wiki/beatmap) that can only be approved.
+A type of [beatmap](/wiki/beatmap) that can only be approved.
 
 Marathon beatmaps are considered to be at least 6 minutes long and can only have one mapped difficulty. However, some marathon mapsets may contain a single [osu!taiko](/wiki/osu!taiko) variant as well.
 
```

Comment:
`Upon the creation of the marathon beatmap, one should name their beatmap version "Marathon" to bypass the [BSS](#bss) from preventing the pending status.`